nautilus_trader/adapters/hyperliquid/data.py

```diff
@@ -55,7 +79,7 @@ class HyperliquidDataClient(LiveMarketDataClient):
     loop : asyncio.AbstractEventLoop
         The event loop for the client.
     client : Any
```

Comment:
We could make this `nautilus_pyo3.HyperliquidHttpClient` now and add to the `nautilus_pyo3.pyi` type stubs (if it's being exported with PyO3).